cmd/download.go

```diff
@@ -205,7 +215,40 @@ func download(folder, release, url, aiInstallerSha, aiAgentSha, aiControllerSha
 		}
 	}
 
-	templatizeImageset(release, folder, aiInstallerSha, aiAgentSha, aiControllerSha, additionalImages)
+	if aiInstallerImage == "" {
```

Comment:
I would move everything to full image location

Reply:
Done